csharp/device/Microsoft.Azure.Devices.Client/Common/SHA.cs

```diff
@@ -0,0 +1,211 @@
+// 
+//  Code by Elze Kool
```

Comment:
I've a reply from Elze stating that "Yes it is public domain. Use it as you like. " 
How should I proceed next?